### PR TITLE
OCPBUGS-17472: Expand regex for fcos/okd packages list

### PR DESCRIPTION
**Pull request by elfosardo**
Exclude also = character in case we need to pin versions for OCP

### CODE REVIEW

File: Dockerfile.fcos [hunk 1]
```diff
@@ -15,7 +15,7 @@ COPY ${PKGS_LIST} ${EXTRA_PKGS_LIST:-$PKGS_LIST} /tmp/
 COPY prepare-image.sh prepare-ipxe.sh configure-nonroot.sh /bin/
 
 # Configure OpenStack repos
-RUN sed -i 's/ >=.*//g' /tmp/main-packages-list.ocp && \
+RUN sed -E -i 's/( =.*| >=.*)//g' /tmp/main-packages-list.ocp && \
```
Comment (dtantsur) [inline Dockerfile.fcos:18]: nit: .* does not have to be inside the brackets

**Comment by elfosardo** [comment]
still seeing the error with the registry on build02
have to wait longer before retesting

**Comment by LorbusChris** [comment]
The failure is unrelated to this change and is being investigated here: https://redhat-internal.slack.com/archives/CEGKQ43CP/p1691583607181299

**Pull request merged**